_partials/_add-data-nyctaxis.mdx

```diff
@@ -0,0 +1,76 @@
+## Load trip data
+
+When you have your database set up, you can load the taxi trip data into the
+`rides` hypertable.
+
+<procedure>
+
+### Loading trip data
+
+<highlight type="important">
+This is a large dataset, so it might take a long time, depending on your network
+connection.
+</highlight>
+
+1.  Download the dataset:
+
+   <tag type="download">
+   [nyc_data.tar.gz](https://timescaledata.blob.core.windows.net/datasets/nyc_data.tar.gz)
+   </tag>
+
+1.  Use your file manager to decompress the downloaded dataset, and take a note
+    of the path to the `nyc_data_rides.csv` file.
+
+1.  At the psql prompt, copy the data from the `nyc_data_rides.csv` file into
+    your hypertable. Make sure you point to the correct path, if it is not in
+    your current working directory:
+
+    ```sql
+    \COPY rides FROM nyc_data_rides.csv CSV;
+    ```
+
+1.  **OPTIONAL:** If you want a faster copy, you can use the TimescaleDB
+   [parallel COPY command][parallel-copy] instead:
+
+    ```bash
+    timescaledb-parallel-copy --connection {CONNECTION STRING} \
+    --db-name {DATABASE NAME} --table rides \
+    --file {PATH TO `nyc_data_rides.csv`} --workers 4 --truncate \
+    --reporting-period 30s
+    ```
```

Comment:
We don't need this. Because for the connection string the information is available in the cheatsheet. The users may not know the values to provide for connection string.